website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+
+    With two types of jobs, instead of one generic type, we can better guide you through the setup flow. Best practices are built into the default settings so you can go from curious to being set up in seconds.
+
+    <Lightbox src="/img/docs/release-notes/ci-job-setup.gif" width="60%" title="Example of setting up a CI job"/>
+
+    And, we now have more efficient state comparisons on CI checks: never waste a build or test on code that hasn’t been changed. We now diff between the Git pull request (PR) code and what’s running in production more efficiently with the introduction of deferral to an environment versus a job. To learn more, refer to [Continuous integration in dbt Cloud](/docs/deploy/continuous-integration). 
+
+    Below is a comparison table that describes how deploy jobs and CI jobs behave differently:
+
+    |  | Deploy Jobs | CI Jobs |
+    | --- | --- | --- |
+    | Purpose | Builds production data assets. | Builds and tests new code before merging changes into production. |
+    | Trigger types | Triggered by a schedule or by API. | Triggered by a commit to a PR or by API. |
+    | Destination | Builds into a production database and schema. | Builds into a staging database and ephemeral schema, lived for the lifetime of the PR. |
+    | Execution mode | Runs execute sequentially, so as to not have collisions on the underlying DAG. | Runs execute in parallel to promote team velocity. |
+    | Efficiency run savings | Detects over-scheduled jobs and cancels unnecessary runs to avoid queue clog. | Cancels existing runs when a newer commit is pushed to avoid redundant work. |
+    | State comparison | Only sometimes needs to detect state. | Almost always needs to compare state against the production environment to build on modified code and its dependents. |
+
+    ## What you need to update
+
+    - If you want to set up a CI environment for your jobs, dbt Labs recommends that you create your CI job in a dedicated [deployment environment](/docs/deploy/deploy-environments#create-a-deployment-environment) that's connected to a staging database. To learn more about these environment best practices, refer to the guide [Get started with continuous integration tests](/guides/set-up-ci).
+
+    - If you had set up a CI job before October 2, 2023, the job might've been misclassified as a deploy job with this update. Below describes how to fix the job type:
+
+        If you used the [Create Job](/dbt-cloud/api-v2#/operations/Create%20Job) API endpoint but didn't set `"triggers":triggers.git_provider_webhook`, the job was misclassified as a deploy job and you must re-create it as described in [Trigger a CI job with the API](/docs/deploy/ci-jobs#trigger-a-ci-job-with-the-api).
```

Comment:
this is also rendering weird -- might be the extra spaced?:

![Screenshot 2024-05-15 at 12 25 53](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/d2fe4d8d-2e6f-42b5-a633-6e4753fff941)